app/controllers/api/v1/users_controller.rb

```diff
@@ -3,6 +3,12 @@
 module Api
   module V1
     class UsersController < ApplicationController
+      before_action :authorization, only: [:index]
+
+      def index
+        @users = User.all
+        render json: @users
```

Comment:
Don't forget to implement `jsonapi-serializer` to show output data.

Reply:
Listo, revisado e implementado